sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/CustomFormModel.java

```diff
@@ -0,0 +1,142 @@
+     */
+    private final List<TrainingDocumentInfo> trainingDocuments;
+
+    /**
+     * Constructs a CustomFormModel object.
+     *
+     * @param modelId Model identifier.
+     * @param modelStatus Status of the model.
+     * @param createdOn Date and time (UTC) when the model was created.
+     * @param lastUpdatedOn Date and time (UTC) when the status was last updated.
+     * @param subModels List of sub models.
+     * @param modelError List of errors returned during the training operation.
+     * @param trainingDocuments List of the documents used to train the model.
+     */
+    public CustomFormModel(final String modelId, final ModelTrainingStatus modelStatus,
+        final OffsetDateTime createdOn, final OffsetDateTime lastUpdatedOn,
+        final IterableStream<CustomFormSubModel> subModels, final List<FormRecognizerError> modelError,
+        final List<TrainingDocumentInfo> trainingDocuments) {
+        this.modelId = modelId;
+        this.modelStatus = modelStatus;
+        this.createdOn = createdOn;
+        this.lastUpdatedOn = lastUpdatedOn;
+        this.subModels = subModels;
+        this.modelError = modelError;
+        this.trainingDocuments = trainingDocuments;
+    }
+
+    /**
+     * Get the modelId property: Model identifier.
+     *
+     * @return the modelId value.
+     */
+    public String getModelId() {
+        return this.modelId;
+    }
+
+    /**
+     * Get the status property: Status of the model.
+     *
+     * @return the status value.
+     */
+    public ModelTrainingStatus getModelStatus() {
+        return this.modelStatus;
+    }
+
+    /**
+     * Get the createdDateTime property: Date and time (UTC) when the model was
+     * created.
+     *
+     * @return the createdDateTime value.
+     */
+    public OffsetDateTime getCreatedOn() {
+        return this.createdOn;
+    }
+
+    /**
+     * Get the lastUpdatedDateTime property: Date and time (UTC) when the
+     * status was last updated.
+     *
+     * @return the lastUpdatedDateTime value.
+     */
+    public OffsetDateTime getLastUpdatedOn() {
+        return this.lastUpdatedOn;
+    }
+
+    /**
+     * Get the errors property: Errors returned during the training operation.
+     *
+     * @return the errors value.
+     */
+    public List<FormRecognizerError> getModelError() {
+        return this.modelError;
+    }
+
+    /**
+     * Get the sub models property: The sub models returned during the training operation.
+     *
+     * @return the sub models value.
+     */
+    public IterableStream<CustomFormSubModel> getSubModels() {
+        return this.subModels;
+    }
+
+    /**
+     * Get the trainingDocuments property: List of the documents used to train
+     * the model and any errors reported in each document.
+     *
+     * @return the trainingDocuments value.
+     */
+    public List<TrainingDocumentInfo> getTrainingDocuments() {
+        return this.trainingDocuments;
+    }
```

Comment:
Why do we have IterableStream in one and List in another as return type?

Reply:
updated both to be of type `IterableStream`